packages/@unimodules/react-native-adapter/src/SyntheticPlatformEmitter.web.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * This emitter is used for sending synthetic native events to listeners
+ * registered in the API layer with `NativeEventEmitter`.
+ */
+export { default } from 'react-native-web/dist/vendor/react-native/NativeEventEmitter/RCTDeviceEventEmitter';
```

Comment:
Added this [upstream](https://github.com/necolas/react-native-web/commit/9ce2b5bf0cfa3e985c669bc3042fd326df965c67#diff-cfc4715d4edc17b80ad19de4a0c4880459b2178cda19300e936790996652c57b) a while ago.